Cauchy sequences in metric spaces (#1347)
UniMath/agda-unimath: src/metric-spaces/cauchy-sequences-complete-metric-spaces.lagda.md

# Cauchy sequences in complete metric spaces

```agda
{-# OPTIONS --lossy-unification #-}

module metric-spaces.cauchy-sequences-complete-metric-spaces where
```

<details><summary>Imports</summary>

```agda
open import foundation.dependent-pair-types
open import foundation.functoriality-dependent-pair-types
open import foundation.universe-levels

open import metric-spaces.cauchy-approximations-metric-spaces
open import metric-spaces.cauchy-sequences-metric-spaces
open import metric-spaces.complete-metric-spaces
open import metric-spaces.convergent-cauchy-approximations-metric-spaces
open import metric-spaces.metric-spaces
```

</details>

## Idea

> A [Cauchy sequence](metric-spaces.cauchy-sequences-metric-spaces.md) in a
> [complete metric space](metric-spaces.complete-metric-spaces.md) is a Cauchy
> sequence in the underlying metric space, and always has a limit.

Reviewer: ```suggestion
A [Cauchy sequence](metric-spaces.cauchy-sequences-metric-spaces.md) in a
[complete metric space](metric-spaces.complete-metric-spaces.md) is a Cauchy
sequence in the underlying [metric space](metric-spaces.metric-spaces.md).
Cauchy sequences in complete metric spaces always have a limit.
```
